cmd/crc/cmd/root.go

```diff
@@ -3,9 +3,14 @@ package cmd
 import (
 	"github.com/spf13/cobra"
 	"os"
+	"strings"
 
+	cmdConfig "github.com/code-ready/crc/cmd/crc/cmd/config"
+	"github.com/code-ready/crc/pkg/crc/config"
+	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/output"
+	"github.com/spf13/viper"
```

Comment:
what is still `viper` that we use in this file?
part of the initi... right? what actually happens here:

  * we set the configfile
  * we set a prefix for the replacer
  * load the config

can this we moved perhaps?

Reply:
Yes, i was actually thinking of putting it in `pkg/crc/config/viper_config.go`